appendix-11-asking for help.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: "Asking for help"
+chapter: "Chapter A10:"
+prev_link: "← Holidays"
+prev_url: "/appendix-09-holidays"
+---
+
+As a fully distributed team, remote calls our lifeblood for our communication. Simple joining a remote team doesn't make you great at communicating remotely, though, and at some point, we're all going to need help. So who do we turn to? How do we ask? Here are a few essential tips we've developed to successfully ask for help within a remote team.
```

Comment:
> remote calls our lifeblood

should this be "remote calls are our lifeblood"?

> Simple joining a remote team

should this be "Simply joining"?

Reply:
I fixed these.